kubernetes/src/main/java/io/kubernetes/client/ApiClient.java

```diff
@@ -90,6 +90,7 @@ public ApiClient() {
         // Setup authentications (key: authentication name, value: authentication).
         authentications = new HashMap<String, Authentication>();
         authentications.put("BearerToken", new ApiKeyAuth("header", "authorization"));
+        authentications.put("BasicAuth", new HttpBasicAuth());
```

Comment:
BTW because `authentications` is private, and there is no way to modify it we (for now) need to make this edit in place. I'll make an edit in swagger codegen that allows us to specify custom authentications.